department-of-veterans-affairs/va.gov-cms
# VACMS-18819: Adjust featured Stories UI

## Description

Relates to #18819

## Testing done

Manual in Tugboat

## Screenshots

![Screenshot 2024-09-30 at 12 18 45 PM](https://github.com/user-attachments/assets/d87cdbcd-1c44-4086-8db7-dee8b54e2c57)

![Screenshot 2024-09-30 at 12 18 51 PM](https://github.com/user-attachments/assets/4dec919b-07d5-41d0-bc35-79a0fb00ea57)


## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As a Content Admin (e.g. david.pickett@va.gov)
1. Log into the [review instance tugboat](https://pr19257-wr6klzocavnn3ujvqlmhiguctu3qci1t.ci.cms.va.gov)
2. Go to Content > Add Content > [Story](https://pr19257-wr6klzocavnn3ujvqlmhiguctu3qci1t.ci.cms.va.gov/node/add/news_story)
   - [ ] Scroll down to the Feature this story box
   - [ ] Notice that the help text for the Featured checkbox is updated 
   - [ ] Notice that the Order dropdown menu no longer appears
3.  Go to Content > Add Content > [Event](https://pr19257-wr6klzocavnn3ujvqlmhiguctu3qci1t.ci.cms.va.gov/node/add/event)
    - [ ] Expand the Feature this Content accordion in the Right Sidebar
    - [ ] Notice that the Order dropdown menu no longer appears
4. View (but don't edit) the following Stories
   - [Five VA Bedford Veterans represent at National Veterans Golden Age Games](https://pr19257-wr6klzocavnn3ujvqlmhiguctu3qci1t.ci.cms.va.gov/bedford-health-care/stories/five-va-bedford-veterans-represent-at-national-veterans-golden-age-games) 
   - [Empowering Those Who Serve Our Veterans](https://pr19257-wr6klzocavnn3ujvqlmhiguctu3qci1t.ci.cms.va.gov/bedford-health-care/stories/empowering-those-who-serve-our-veterans)
   - [Hospital receives bivalent booster vaccines](https://pr19257-wr6klzocavnn3ujvqlmhiguctu3qci1t.ci.cms.va.gov/bedford-health-care/stories/hospital-receives-bivalent-booster-vaccines)
   - Noticed that all these Stories are set as Featured
   - Edit one of the Stories and save it as Featured
   - Check the other Stories and confirm they are now set as Not Featured
5. View (but don't edit) the following Events
   - [VA Clinic at Vinita Health Center Grand Opening](https://pr19257-wr6klzocavnn3ujvqlmhiguctu3qci1t.ci.cms.va.gov/eastern-oklahoma-health-care/events/69209)
   - [Spring Planting Workshop](https://pr19257-wr6klzocavnn3ujvqlmhiguctu3qci1t.ci.cms.va.gov/eastern-oklahoma-health-care/events/55255)
   - [PACT Act Claims Event](https://pr19257-wr6klzocavnn3ujvqlmhiguctu3qci1t.ci.cms.va.gov/eastern-oklahoma-health-care/events/66516)
   - Scroll down to the Curation box and noticed that all these events are set as Featured: On
   - Edit one of these Events and save it as Featured
   - Check the other Events and confirm they now are set as Featured: Off

### Definition of Done
